src/block_cache/block_chunkcache.c

```diff
@@ -510,15 +559,37 @@ __wt_chunkcache_setup(WT_SESSION_IMPL *session, const char *cfg[], bool reconfig
         chunkcache->type = WT_CHUNKCACHE_IN_VOLATILE_MEMORY;
     else if (WT_STRING_MATCH("file", cval.str, cval.len) ||
       WT_STRING_MATCH("FILE", cval.str, cval.len)) {
-#ifdef ENABLE_MEMKIND
+
         chunkcache->type = WT_CHUNKCACHE_FILE;
         WT_RET(__wt_config_gets(session, cfg, "chunk_cache.device_path", &cval));
         WT_RET(__wt_strndup(session, cval.str, cval.len, &chunkcache->dev_path));
         if (!__wt_absolute_path(chunkcache->dev_path))
             WT_RET_MSG(session, EINVAL, "File directory must be an absolute path");
-#else
-        WT_RET_MSG(session, EINVAL, "chunk cache of type FILE requires libmemkind");
-#endif
+
+        WT_RET(__wt_open(session, chunkcache->dev_path, WT_FS_OPEN_FILE_TYPE_REGULAR,
+          O_RDWR | O_TRUNC | O_CREAT, &fh));
+
+        WT_RET(
+          fh->handle->fh_truncate(fh->handle, &session->iface, (wt_off_t)chunkcache->capacity));
+        // ret = truncate(chunkcache->dev_path, );
+        // if (ret != 0)
+        // WT_RET_MSG(session, ret, "Error truncating file to size %lu \n", chunkcache->capacity);
+
+        /* Allocate memory for the chunk cache for type file system. */
+        WT_RET(fh->handle->fh_map(
+          fh->handle, &session->iface, (void **)&chunkcache->memory, &mapped_size, NULL));
```

Comment:
We should check that `mapped_size == chunkcache->capacity` somewhere.

Reply:
done here [0366ff8](https://github.com/wiredtiger/wiredtiger/pull/9442/commits/0366ff84099f6d54731d63f524f8756b77f551e1)

Reply:
done here: [690e97d](https://github.com/wiredtiger/wiredtiger/pull/9442/commits/690e97db34d0ac5088874a15cf526129f4065290)